### PR TITLE
Update S3 uploader to use signed URL, close #70

### LINKED ISSUE #70: Update S3 uploader to use signed URL
**awsUploader**:

- Request signed URL
  ```json
  {
    "serviceName": "dpe",
    "fileName": "users/id/audio/blah.wav",
    "duration": 60
  }
  ```
- PUT file to signed url

**sttChecker**:

```json
{
  "serviceName": "dpe",
  "fileName": "users/id/audio/blah.wav"
}
```

### PR DESCRIPTION
**Pull request by bennuttall**
**Is your Pull Request request related to another [issue](https://github.com/bbc/react-transcript-editor/issues) in this repository ?**      

- Closes #70

**Describe what the PR does**    

- Update S3 uploader to use signed URL
- Update sttChecker to send serviceName and fileName instead of objectKey

**State whether the PR is ready for review or whether it needs extra work**    

Needs testing

### CODE REVIEW

File: functions/awsUploader/index.js [hunk 1]
```diff
@@ -1,20 +1,35 @@
...
+      if (err) {
+        console.log("err", err);
+        reject(err)
+      }
+
+      console.log("url", url);
+      resolve(url);
+    })
   });
+}
```
Comment (emettely) [inline functions/awsUploader/index.js:17]: this looks good - can it be moved to utils?

File: functions/awsUploader/index.js [hunk 2]
```diff
@@ -1,20 +1,35 @@
...
+  const promise = fetch(url, params);
+
   return {
     writeStream: pass,
-    promise: s3.upload({ Bucket, Key, Body: pass, Metadata }).promise(),
-  };
-};
+    promise: promise,
+  }
+}
```
Comment (emettely) [inline functions/awsUploader/index.js:32]: same with this :) move to utils, looks like a function that can be reused

File: functions/sttChecker/index.js [hunk 3]
```diff
@@ -109,14 +112,19 @@ const updateTranscriptsStatus = async (
 
   await validJobs.forEach(async (job) => {
     let response;
-    const userId = usersAudioData[job.id]["user"];
-    const objectKey = `dpe/users/${userId}/audio/${job.id}.wav`;
+    const usersAudioDataJob = usersAudioData[job.id];
+    if (!usersAudioDataJob) {
+      console.log(`[ERROR] Job ID {job.id} not found`);
```
Comment (emettely) [inline functions/sttChecker/index.js:117]: 👍 thanks

File: functions/sttChecker/index.js [hunk 4]
```diff
@@ -137,7 +145,7 @@ const updateTranscriptsStatus = async (
 
     const { projectId } = job.data();
     await updateTranscription(admin, job.id, projectId, update);
-    console.debug(`Updated ${job.id} with data`, update);
+    // console.debug(`Updated ${job.id} with data`, update);
```
Comment (emettely) [inline functions/sttChecker/index.js:148]:
```suggestion
    console.debug(`Updated ${job.id} with data`, update);
```

File: functions/sttChecker/index.js (hunk 3 above)
Comment (bennuttall) [reply functions/sttChecker/index.js:117]: done

File: functions/awsUploader/index.js (hunk 2 above)
Comment (bennuttall) [reply functions/awsUploader/index.js]: done

File: functions/awsUploader/index.js (hunk 1 above)
Comment (bennuttall) [reply functions/awsUploader/index.js]: done

**Pull request merged**